doc/source/tutorial/getting_started.rst

```diff
@@ -278,6 +291,14 @@ could be used instead by add ``++local`` to ``charmrun``, e.g.:
 
      ``~/Charm/bin/charmrun ++local +p4 bin/enzo-p input/HelloWorld/Hi.in``
 
+If you receive an error like
+
+.. code-block:: bash
+    Charmrun> Timeout waiting for node-program to connect
```

Comment:
```suggestion
..  code-block:: bash

    Charmrun> Timeout waiting for node-program to connect
```

This might help with a formatting issue on the website build from the test